### ISSUE TITLE
Ability to handle folder paths in BACKUP_FILENAME

### ISSUE THREAD

**Issue by michael-robbins** [jareware/docker-volume-backup#22]
It would be nice to be able to handle the BACKUP_FILENAME containing a relative folder path.

In combination with uploading to S3 this would let me upload into a folder structure.

**Comment by OMGTheCloud** [comment]
Old issue, but this is working at least as of now: if you specify a bucket + path in the `AWS_S3_BUCKET_NAME` env var, it will create that folder (if it doesn't exist) and store the named backup file in it. Note: the bucket itself *does* have to exist! but the *folder* will be created.

Example:
```
    environment:
      AWS_S3_BUCKET_NAME: "your-bucketname/wordpress-sites"
      BACKUP_FILENAME: wp-backup-latest.tar.gz
```
... will create `wordpress-sites` folder under `your-bucketname`, and place your `wp-backup-latest.tar.gz` in it.

**Comment by jareware** [comment]
Thanks for pointing this out, I've now included this tip in the docs (b2ee77ce14ed61796d5cc5285dde46ddfbed8870)!
